src/gpu/intel/gemm/jit/gen_kernel.hpp

```diff
@@ -97,6 +107,7 @@ struct gen_desc_t {
     const gemmstone::kcatalog::Entry *entry_ = nullptr;
     gemmstone::EvaluateAuxOutput aux_params_;
     gemmstone::CommonDriverInfo driver_info_;
+    compute::kernel_t kernel_ = nullptr;
```

Comment:
Default constructor is enough.

```suggestion
    compute::kernel_t kernel_;
```